src/argv.ts

```diff
@@ -225,6 +226,7 @@ export class Argv {
     }
 
     get artifactsToSource (): boolean {
+        // TODO: default to false in 5.x.x
```

Comment:
Is this a typo?

Reply:
artifacts to source will be default "off" in 5.x.x